x-pack/filebeat/module/mssql/log/ingest/pipeline.json

```diff
@@ -0,0 +1,50 @@
+{
+    "description": "Pipeline to parse MSSQL logs",
+    "processors": [
+        {
+            "grok": {
+                "field": "message",
+                "patterns": ["%{MSSQL_DATE:date} %{DATA:mssql.log.origin} [ ]*%{GREEDYDATA:msg_temp}"],
+                "pattern_definitions": {
+                    "MSSQL_DATE":"%{DATA} %{DATA}"
+                }
+            }
+        },
+        {
+            "date": {
+                "field": "date",
+                "target_field": "@timestamp",
+                "formats": ["yyyy-MM-dd HH:mm:ss.SS"],
+                {< if .convert_timezone >}"timezone": "{{ event.timezone }}",{< end >}
```

Comment:
Based on the recent changes from @jsoriano we should use the if statement here?

Reply:
+1, this is the change @ruflin is talking about: https://github.com/elastic/beats/pull/12311

I'm good with doing it here or merging now + open a follow up

Reply:
Added!